specs/swagger.yaml

```diff
@@ -289,7 +299,7 @@ paths:
      get:
       summary: Get a collection of peptides   
       operationId: getPeptides
-      description: The peptide entrey point returns a global Peptidoform statistics across an entire resource. Each peptide contains a summary of the statistics of the peptidoform across the entire resource.  
+      description: The peptide entrey point returns a global peptidoform statistics across an entire resource. Each peptide contains a summary of the statistics of the peptidoform across the entire resource.
```

Comment:
1. Misspelled "entry".
2.  Remove "a" since "global peptidoform statistics" is plural.